site/src/tokens/colors.md

```diff
@@ -9,29 +9,98 @@ token: colors
 
 Atlas currently supports three themes out of the box.
 
-## Background Colors
+## Stops
 
-<div class="padding-xl background-color-body">
-	<p class="font-weight-bold font-size-h4 margin-bottom-xs">body</p>
-	<p><code>.background-color-body</code></p>
+<div class="padding-top-sm margin-bottom-md">
+Atlas colors are available in a number of stops from dark to light—100 being the darkest, and 10 being the lightest.
 </div>
 
-<div class="padding-xl background-color-body-medium">
-	<p class="font-weight-bold font-size-h4 margin-bottom-xs">body-medium</p>
-	<p><code>.background-color-body-medium</code></p>
+<div class="display-flex gap-xxs">
+	<div class="padding-md background-color-black-static color-text-overlay-invert width-full border-radius">
+		<p class="font-weight-bold font-size-h4 margin-bottom-xs">Black Static</p>
+		<p class="font-weight-bold font-size-h6 margin-bottom-xs">aka “Black”</p>
+		<p><code>#000000</code></p>
+	</div>
+	<div class="padding-md background-color-white-static color-black-static width-full border-radius border--color-black">
+		<p class="font-weight-bold font-size-h4 margin-bottom-xs">White</p>
+		<p class="font-weight-bold font-size-h6 margin-bottom-xs">aka "White"</p>
+		<p><code>#FFF</code></p>
+	</div>
 </div>
 
-<div class="padding-xl background-color-body-accent">
-	<p class="font-weight-bold font-size-h4 margin-bottom-xs">body-accent</p>
-	<p><code>.background-color-body-accent</code></p>
-</div>
+<div>
+
+### Primary colors
+
+Approved background colors.
 
-<div class="padding-xl background-color-alternate color-text-invert">
-	<p class="font-weight-bold font-size-h4 margin-bottom-xs">alternate</p>
-	<p><code>.background-color-alternate</code></p>
 </div>
 
-<div class="padding-xl background-color-alternate-medium color-text-invert margin-bottom-xl">
-	<p class="font-weight-bold font-size-h4 margin-bottom-xs">alternate-medium</p>
-	<p><code>.background-color-alternate-medium</code></p>
+<div class="display-flex gap-xxs">
+	<div class="width-full">
+		<h4 class="font-weight-bold font-size-h4 margin-bottom-xs"> Primary colors</h4>
+		<p>Approved background colors.</p>
+		<div class="display-flex">
+			<div id="blueList" class="display-inline-block"></div>
+			<div id="navyList" class="display-inline-block"></div>
+			<div id="greyList" class="display-inline-block"></div>
+		</div>
+	</div>
+	<div class="width-full">
+		<h4 class="font-weight-bold font-size-h4 margin-bottom-xs"> Secondary colors</h4>
+		<p>Secondary colors are used only for alerts.</p>
+		<div class="display-flex">
+			<div id="turquoiseList" class="display-inline-block"></div>
+			<div id="yellowList" class="display-inline-block"></div>
+			<div id="greenList" class="display-inline-block"></div>
+			<div id="redList" class="display-inline-block"></div>
+			<div id="purpleList" class="display-inline-block"></div>
+		</div>
+	</div>
 </div>
+
+## Theming
```

Comment:
Themed colors and their corresponding classes are the most essential documentation for improving our developer experience. Thinking these should be the main showcase of the "colors" page, while the above "Stops" section should be moved to palette.

The essential variables of themed content are the following (pulled from /tokens/themes)

Standard text colors:
		'text': $palette-grey-30,
		'text-subtle': $palette-grey-40,
		'text-invert': $palette-grey-110,
		'visited': $palette-purple-40,
		'hyperlink': $palette-blue-30,

Standard background colors
		'body-background': $palette-grey-120,
		'body-background-accent': $palette-blue-black,
		'body-background-medium': $palette-grey-110,
		'alternate-background': $palette-grey-20,
		'alternate-background-medium': $palette-grey-10,

Border colors
		'border': $palette-grey-100,
		'border-accent': $palette-grey-90,
		'border-white-high-contrast': $palette-white-opacity-0,
		'border-yellow-high-contrast': $palette-black-opacity-0,

Now the most important semantic colors: primary, secondary, success, info, danger. All of which have the same several properties with their names prepended.

		'primary-base': $palette-blue-30,
		'primary-background': $palette-blue-80,
		'primary-background-glow-high-contrast': $palette-blue-80,
		'primary-dark': $palette-blue-20,
		'primary-hover': $palette-blue-40,
		'primary-active': $palette-blue-50,
		'primary-box-shadow': $palette-blue-opacity-30,
		'primary-invert': $palette-black,

		'secondary-base': $palette-grey-80,
		'secondary-background': $palette-grey-100,
		'secondary-background-glow-high-contrast': $palette-grey-100,
		'secondary-dark': $palette-grey-30,
		'secondary-hover': $palette-grey-70,
		'secondary-active': $palette-grey-60,
		'secondary-box-shadow': $palette-black-opacity-70,
		'secondary-invert': $palette-white,

		'success-base': $palette-green-30,
		'success-background': $palette-green-80,
		'success-background-glow-high-contrast': $palette-green-80,
		'success-dark': $palette-green-10,
		'success-hover': $palette-green-40,
		'success-active': $palette-green-30,
		'success-box-shadow': $palette-green-opacity-70,
		'success-invert': $palette-black,

		'info-base': $palette-purple-50,
		'info-background': $palette-purple-80,
		'info-background-glow-high-contrast': $palette-purple-80,
		'info-dark': $palette-purple-10,
		'info-hover': $palette-purple-40,
		'info-active': $palette-purple-30,
		'info-box-shadow': $palette-purple-opacity-70,
		'info-invert': $palette-black,

		'danger-base': $palette-red-20,
		'danger-background': $palette-red-80,
		'danger-background-glow-high-contrast': $palette-red-80,
		'danger-dark': $palette-red-10,
		'danger-hover': $palette-red-40,
		'danger-active': $palette-red-30,
		'danger-box-shadow': $palette-red-opacity-70,
		'danger-invert': $palette-black,

Needn't be done as part of this PR, but I think documenting the above is the most important gap we have in our current documentation.

Reply:
One nice thing about all the properties above is that by appending `--theme-` we have a usable css variables.

--theme-danger-base
--theme-info-invert
--theme-border
--theme-text
--theme-body-background